war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -500,16 +500,49 @@ var tooltip;
 // Behavior rules
 //========================================================
 // using tag names in CSS selector makes the processing faster
+
+
+/**
+ * Updates the validation area for a form element
+ * @param {HTMLElement} validationArea The validation area for a given form element
+ * @param {string} content The content to update the validation area with
+ */
+function updateValidationArea(validationArea, content) {
+  validationArea.classList.add("validation-error-area--visible");
+
+  if (content === "<div/>") {
+    validationArea.classList.remove("validation-error-area--visible");
+    validationArea.style.height = "0px";
+    validationArea.innerHTML = content;
+  } else {
+    // Only change content if different, causes an unnecessary animation otherwise
+    if (validationArea.innerHTML !== content) {
+      validationArea.innerHTML = content;
+      validationArea.style.height = validationArea.children[0].offsetHeight + "px";
+
+      // Only include the notice in the validation-error-area, move all other elements out
+      if (validationArea.children.length > 1) {
+        Array.from(validationArea.children).slice(1).forEach((element) => {
+          validationArea.after(element);
+        })
+      }
+
+      Behaviour.applySubtree(validationArea);
```

Comment:
The change in https://github.com/jenkinsci/jenkins/pull/6460/commits/42cac629598d1f7f725a787aa178264a63e5622e broke expanding stacktraces.

A trivial way to reproduce is with a GitHub App credential from github-branch-source plugin, just put gibbish in the private key field

Reply:
Should be fixed now - thanks for spotting! :)